Scruffy.Services/Discord/DialogSelectMenuElementBase.cs

```diff
@@ -37,6 +37,12 @@ protected DialogSelectMenuElementBase(LocalizationService localizationService)
     /// <returns>Message</returns>
     public abstract string GetMessage();
 
+    /// <summary>
+    /// Whether the return message is ephemeral
+    /// </summary>
+    /// <returns>Ephemeral</returns>
+    public virtual bool IsEphemeral() => false;
```

Comment:
Can be declared in `DialogElementBase`. Then you can use the method in `DialogButtonElementBase` to make `RaidRoleSelectionNextDialogElement` also ephemeral.